### ISSUE TITLE
Add in FAQ

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#695]
Add in FAQ (based on this http://www.cs.umd.edu/~jonf/advising.html) that also has a ugrad research apply link.

Update: now that @htadeusiak is going to work on this, I want to be more clear what I was envisioning here. I want to base the general design on this page https://sidewalk.cs.washington.edu/faq and https://sidewalk-test.cs.washington.edu/labelingGuide where we have a floating left menu bar of questions (should be hierarchical with an expanding menu--otherwise, the floating menu will be overwhelming).

![image](https://user-images.githubusercontent.com/1621749/50231710-6ca41a00-037d-11e9-99e2-a5adfebfd68e.png)

**Comment by htadeusiak** [comment]
Here is my first draft of the FAQ. I tried copying the format of the labeling guide from Project Sidewalk. Correct me if I am wrong, but the labeling guide would open a new page every time you chose a different obstacle from the panel. I feel like that is slightly inefficient so I removed it from my solution. The webpage will only load once at the beginning.

I have not done any CSS to it so it's a bit bare. But, I wanted to check with you to see if this is the functionality you want out of this page before I continue.

![kapture 2018-12-20 at 16 54 45](https://user-images.githubusercontent.com/33988444/50318764-22ec2a00-0478-11e9-9151-c688a9758d41.gif)

**Comment by jonfroehlich** [comment]
Woohoo! Looks great. I think we should have a header on the FAQ (to make us feel more accessible and fun--maybe the same header as the people page; I can help with this but just put the standard header code in there).

I'll obviously have to edit some of the text because it's old and written for UMD but we can do that after we finalize on the design and get the PR merged on makeability-test. 

Could you try to reduce the indentation of the questions under the sub-headers. Too much whitespace on the left.

**Comment by htadeusiak** [comment]
This example is what would be in the "content" of the page. So yes header and footer will be there.

I know that all the pages extend the base.html page, can we do that with this page? Or should I just copy the header and footer code from that page into this one?

I'll fix the indentation.

**Comment by jonfroehlich** [comment]
Just extend base.html--that should be fine.

And then I agree with you that eventually we will want to Django'fy this (will want to create another Issue for it) so that we can add and edit content via the Django admin interface. However, before we do so, I want to find a rich text editor plugin for Django. Can you make this new issue?

**Comment by htadeusiak** [comment]
Added the banner, header, and footer through extend base.html. Adjusted the white space indents.

I also added some css transitions to the panel guide. I can refine them more, but wanted your opinion on if you wanted them for this page.

Is there anything else that you wanted for this page I missed?

![faq2](https://user-images.githubusercontent.com/33988444/50389001-9ab88f80-06d8-11e9-8076-ffbc6c2286c6.gif)
